_dashboards/management/multi-data-sources.md

```diff
@@ -141,9 +143,65 @@ data_source.hideLocalCluster: true
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/multidata-hide-localcluster.gif" alt="Multiple data sources hide local cluster">{: .img-fluid}
 
+## Using multiple data sources with external dashboards plugins
+Introduced 2.14
+{: .label .label-purple }
+
+The following plugins now support multiple data sources
+
+### Index management
+
+When the data source feature is enabled, you can navigate to **Index Management** under the **Management** menu. Using indexes as an example, you can view all connected data sources and select a specific one from the navigation bar at the upper right. By default, the indexes from the designated default data source is selected are displayed. However, you can select any connected data source to view its corresponding indexes. The following GIF illustrates these steps.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ism_mds1.gif" alt="Multiple data sources in ISM list page"/>
+
+To perform operations on a specific index within a data source, select the individual index from the list. To create a new index, select the `Create Index` button, which opens a form. Fill in the required information and select the `Create` button. The index is created within the selected data source. The following GIF illustrates these steps. 
```

Comment:
Should the names of the buttons be bold instead of in code font?

Reply:
sure, change name of button to bold in new commit

Reply:
@natebower @BionIT I thought I changed them all. I'll format.